### PR TITLE
chore(CommunitySettings): Remove old legacy community popup

### PR DESCRIPTION
**Pull request by borismelnik**
Fixes: #9913

### What does the PR do

Remove legacy community popup call from settings.

PS: Don't know the popup should be removed as well cos it used in another place.  

### Affected areas

Community Settings

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### CODE REVIEW

**Review by caybro — CHANGES_REQUESTED** [review]
How do we transfer ownership now or share the link? This probably needs some design clarification first. It is still accessible via a context menu in the primary nav but this is not optimal (should be also accessible somewhere from the community settings)

talking about this:
image

**Comment by jrainville** [comment]
@caybro it's still accessible from the right click menu on the main nav (View community)

image

We need some UX improvements in a couple places, but at least it's still available :smile:

File: ui/app/mainui/Popups.qml [hunk 1]
```diff
@@ -33,7 +33,6 @@ QtObject {
         Global.openBackUpSeedPopup.connect(openBackUpSeedPopup)
         Global.openEditDisplayNamePopup.connect(openEditDisplayNamePopup)
         Global.openPinnedMessagesPopupRequested.connect(openPinnedMessagesPopup)
-        Global.openCommunityProfilePopupRequested.connect(openCommunityProfilePopup)
```
Comment (caybro) [inline ui/app/mainui/Popups.qml:36]: This must stay too

**Review by caybro — CHANGES_REQUESTED** [review]
Fine then but still needs fixes, if it's meant to stay available via the primary nav

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 2]
```diff
@@ -125,46 +124,23 @@ StatusSectionLayout {
...
-                font.pixelSize: 15
-                font.underline: true
-
-                MouseArea {
-                    anchors.fill: parent
-                    cursorShape: Qt.PointingHandCursor
-                    onClicked: root.backToCommunityClicked()
-                    hoverEnabled: true
-                }
+            text: "<- " + qsTr("Back to community")
```
[the remark this answers is not in the document]
Comment (borismelnik) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:142]: @caybro could you pls show me an example? where I can find it?

**Comment by caybro** [comment]
Also missing some kind of screenshot/video here, demonstrating the feature still works as expected

File: ui/app/AppLayouts/Chat/ChatLayout.qml [hunk 3]
```diff
@@ -136,8 +136,6 @@ StackLayout {
 
             onBackToCommunityClicked: root.currentIndex = 0
 
-            // TODO: remove me when migration to new settings is done
-            onOpenLegacyPopupClicked: Global.openCommunityProfilePopupRequested(root.rootStore, community, chatCommunitySectionModule)
```
[the remark this answers is not in the document]
Comment (caybro) [reply ui/app/AppLayouts/Chat/ChatLayout.qml:140]: Yeah, the popup is invoked from the primary nav directly via (from AppMain.qml):
```qml
StatusAction {
  text: qsTr("View Community")
  icon.name: "group-chat"
  onTriggered: popups.openCommunityProfilePopup(appMain.rootStore, model, communityContextMenu.chatCommunitySectionModule)
}
```

However, the popup must stay as it gives access to the "transfer" feature:
image

Or do we plan to put this somewhere else?

**Comment by borismelnik** [comment]
@caybro @jrainville guys I totally missed the context... @jrainville asking about removing `openCommunityProfilePopupRequested`, @caybro asking about to leave it... so let's try to setting up the requirements:
1. Popup should be deleted?
2. Every signal and method as well?
3. What should be displayed when we click on `View Community` from context menu?

File: ui/app/AppLayouts/Chat/ChatLayout.qml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jrainville) [reply ui/app/AppLayouts/Chat/ChatLayout.qml:140]: Oh, my bad, I didn't realize it was the same popup/function.
I thought it's what called the opening of the legacy edit popup.

So, does that mean that the old legacy popup is already removed? I checked quickly and it seems so.

Alright then, resolving this comment

File: ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml [hunk 4]
```diff
@@ -125,46 +124,23 @@ StatusSectionLayout {
...
+            color: Theme.palette.baseColor1
+            font.pixelSize: 15
+            font.underline: true
+
+            MouseArea {
```
[the remark this answers is not in the document]
Comment (jrainville) [reply ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml:147]: This one is still used to go back to the community when in Edit mode

**Review by jrainville — APPROVED** [review]

**Comment by caybro** [comment]
The context is clear imho: remove the access to the (old/deprecated) community profile popup:
image

This is the popup (still accessible via context/RMB menu on the primary nav):

image


image

However, we can't simply remove the popup itself too, as we don't have any other way to access the "Transfer ownership" feature, or am I missing something @jrainville ? :)

**Comment by borismelnik** [comment]
@caybro Pls check Jo answer :) lets merge it!

**Pull request merged**